projects/nodejs/build.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Copyright 2017 Google Inc.
+# Modifications copyright (C) 2022 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+set -e
+
+CC=clang
+CXX=clang++
+
+# libFuzzer
+cd /node_libfuzzer
+
+CXXFLAGS="-g -fsanitize=fuzzer-no-link,address"
```

Comment:
Does undefined sanitizer fails to build? We won't be able to verify security predicates (our own symbolic checkers) when undefined sanitizer is disabled.

Reply:
Hi , i need a little bit of time to debug harness with address sanitizer. After that, i'm going to check a harness with undefined sanitizer. 
Previously, i've used AFL++ fuzzer and everything has been alright